apps/analytics/processed_items_report.py

```diff
@@ -0,0 +1,79 @@
+from superdesk import get_resource_service
+from superdesk.services import BaseService
+
+from superdesk.metadata.item import metadata_schema
+from superdesk.resource import Resource
+
+
+class ProcessedItemsResource(Resource):
+    """Processed items schema
+    """
+
+    schema = {
+        'user': metadata_schema['original_creator'],
+        'starting_time': {'type': 'datetime'},
+        'ending_time': {'type': 'datetime'}
+    }
+
+    item_methods = ['GET', 'DELETE']
+    resource_methods = ['POST']
+
+    privileges = {'POST': 'processed_items_report', 'DELETE': 'processed_items_report', 'GET': 'processed_items_report'}
+
+
+class ProcessedItemsService(BaseService):
+
+    def create_query(self, doc):
+        archive_versions_query = {
+            '$or': [{"task.user": str(doc['user'])}]
+        }
+        return archive_versions_query
+
+    def get_items(self, query):
+        """Return the result of the item search by the given query
+        """
+        return get_resource_service('archive_versions').get(req=None, lookup=query)
+
+    def count_items(self, query, state, starting, ending):
+        """
+        Return the number of items which was created in a given time interval and having a certain state.
+        If no state is give, it will return all items created in the given time interval.
+        """
+        my_list = self.get_items(query)
+        if state == '':
+            items = sum(1 for i in my_list
+                        if (i['state'] in ['published', 'spiked', 'corrected', 'killed', 'updated']) and
+                        str(i['firstcreated']) >= str(starting) and str(i['firstcreated']) <= str(ending))
```

Comment:
You should use the _updated field instead of firstcreated. _updated is set for each item version, firstcreated doesn't change. We are only interested in the version timestamp.